pass-nihms-loader/nihms-data-transform-load/src/test/java/org/eclipse/pass/loader/nihms/TransformAndLoadInProcessIT.java

```diff
@@ -199,13 +205,13 @@ public void testInProcessExistingSubmissionDeposit() throws Exception {
 
     }
 
-    private NihmsPublication newInProcessNihmsPub() {
-        String dateval = "12/12/2017";
+    private NihmsPublication newInProcessNihmsPub(String grant1, String pmid1) {
+        String dateval = "12/10/2001";
```

Comment:
I don't think this matters, but is there a reason for testing an older date?

Reply:
I just wanted some different numbers to test. It doesn't really matter.